mmdet/datasets/dataset_wrappers.py

```diff
@@ -401,15 +401,26 @@ def __getitem__(self, idx):
                 continue
 
             if hasattr(transform, 'get_indexes'):
-                indexes = transform.get_indexes(self.dataset)
-                if not isinstance(indexes, collections.abc.Sequence):
-                    indexes = [indexes]
-                mix_results = [
-                    copy.deepcopy(self.dataset[index]) for index in indexes
-                ]
-                results['mix_results'] = mix_results
-
-            results = transform(results)
+                while True:
+                    # RandomCrop and Albu may return None.
```

Comment:
This comment seems incorrect. RandomCrop and Albu do not belong to the load pipeline
'' Make sure the results passed the pipeline of the original dataset is not None.” would be more accurate.